andela-ifatoki/CP2-Shift-DMS
# #148994195 Implement document access features

#### What does this PR do?
It incorporates document access features into the project
#### Description of Task to be completed?
User should be able to make a document Public, private or shared with a role/individual
#### How should this be manually tested?

- After sign in, click on the new document Icon on the top right side of the page. 
- You should get a pop up window giving you an avenue to create a document with radio buttons giving some avenue to allocate access roles to the document.

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
148994195
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
